awx/ui_next/src/components/Lookup/Lookup.jsx

```diff
@@ -145,7 +205,7 @@ class Lookup extends React.Component {
 
     return (
       <Fragment>
-        <InputGroup className="awx-lookup">
+        <InputGroup>
```

Comment:
We should hide the scrollbar on these somehow.

Reply:
Good catch. Added a fix for this.